### PR TITLE
Removes isinstance and issubclass workarounds

### LINKED ISSUE #23: remove isinstance and issubclass workarounds
remove workaround added in #22 after Jep releases https://github.com/ninia/jep/pull/440

### PR DESCRIPTION
**Pull request by colton-gabertan**
This change resolves #23.

I've removed the workarounds and modified the test_jepbridge to reflect the changes. Unit tests seem to be passing on my end!

### CODE REVIEW

File: data/python/tests/test_jepbridge.py [hunk 1]
```diff
@@ -28,10 +28,10 @@ def test_type_instance(self):
         from java.util import Date
         from ghidra.program.database import ProgramDB
 
-        self.assertIsInstance(Date(), Object.__pytype__)
-        self.assertIsInstance(Date(), Serializable.__pytype__)
-        self.assertTrue(issubclass(Date.__pytype__, Object.__pytype__))
-        self.assertTrue(issubclass(Date.__pytype__, Serializable.__pytype__))
```
Comment (mike-hunhoff) [inline data/python/tests/test_jepbridge.py:34]: Was `__pytype__` removed in the latest `Jep` release? If not, let's keep these tests for good measure. If `__pytype__` was removed then let's be sure to de-duplicate these tests.

**Review by mike-hunhoff — CHANGES_REQUESTED** [review]
Thank you @colton-gabertan! I've left one comment to address. Also, can you please update the `README` to include the min. `Jep` version needed for your changes to work?

File: data/python/tests/test_jepbridge.py (hunk 1 above)
Comment (colton-gabertan) [reply data/python/tests/test_jepbridge.py:34]: @mike-hunhoff I've confirmed that the latest `Jep` release still has the `__pytype__` access in their unit test. I'll be adding these back in the next commit and updating the documentation.

**Review by mike-hunhoff — APPROVED** [review]
Nice @colton-gabertan LGTM 🚀

**Pull request merged**